src/actions/ast.js

```diff
@@ -126,3 +138,64 @@ export function setSelection(
     });
   };
 }
+
+/**
+ * Action dispatched from stepOver action. Receives the position
+ * and depending on where it is paused, sets up hidden breakpoints
+ * and handles resume, stepOver (as needed)
+ * @param position
+ * @returns {function(ThunkArgs)}
+ */
+export function analyzeStepping(type) {
+  return async ({ dispatch, getState, client }: ThunkArgs) => {
+    const position = await dispatch(getPosition());
+    const source = getSelectedSource(getState()).toJS();
+    const hiddenBreakpointLocation = getHiddenBreakpoint(getState());
+
+    await dispatch({
```

Comment:
I can probably pull this out of a `dispatch` call as it is not going to hit a reducer. most likely this is switching through reducers and doing a noop as there's no case for it. wonder if there's a better way to handle this?